pilot/pkg/model/context.go

```diff
@@ -25,7 +25,7 @@ import (
 	"strings"
 	"time"
 
-	core "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
+	corev3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
```

Comment:
nit: Should we be consistent with package names? Some places we are calling `enpoints` and place like here we are calling `corev3` - If there is no specific reason to call it `corev3`, can we just call `core` as we were previously?

Reply:
In endpoint case we *only* have `endpoints`, but in core we are still using v2 in many parts of the code which may cause some confusion. Eventually this will be all v3 but will take ~4 PRs or so to avoid having a massive 5k line PR. 

regarding end state - I don't have a strong preference on naming things `foo` or `foov3` so any opinions welcome here